teuthology/lock.py

```diff
@@ -385,6 +405,11 @@ def lock_many(ctx, num, machine_type, user=None, description=None,
     machine_types_list = misc.get_multi_machine_types(machine_type)
     if machine_types_list == ['vps']:
         machine_types = machine_types_list
+    elif machine_types_list == ['openstack']:
```

Comment:
I'm pretty sure things would go sideways if we tried to mix openstack and non openstack machines in the same cluster.